### PR TITLE
WIP: Termcolor 1.0.0

### PR DESCRIPTION
**Pull request by norbertwenzel**
@NeroBurner Could you please take a look before submitting to the main repo. Thanks.

### CODE REVIEW

File: docs/packages/pkg/termcolor.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+.. index::
+  single: unsorted ; termcolor
+
+.. _pkg.termcolor:
+
+termcolor
+=========
+
+-  `Official <https://github.com/ikalnytskyi/termcolor>`__
+-  `Hunterized <https://github.com/hunter-packages/termcolor>`__
```
Comment (NeroBurner) [inline docs/packages/pkg/termcolor.rst]: There will be no hunterized fork. All is upstream

File: docs/packages/pkg/termcolor.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.. spelling::
+
+    termcolor
+
+.. index::
+  single: unsorted ; termcolor
```
Comment (NeroBurner) [inline docs/packages/pkg/termcolor.rst]: I'd put it into the class `commandline tools` or try to find another better describing one https://docs.hunter.sh/en/latest/genindex.html

File: docs/packages/pkg/termcolor.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+.. _pkg.termcolor:
+
+termcolor
+=========
+
```
Comment (NeroBurner) [inline docs/packages/pkg/termcolor.rst:12]: maybe add a short summary what termcolor is. Taken from the project description

> Termcolor is a header-only C++ library for printing colored messages to the terminal. Written just for fun with a help of the Force.

**Review by NeroBurner — COMMENTED** [review]
just some documentation changes

don't forget to create a CI branch based on `pkg.template` branch as described in the documentation
https://docs.hunter.sh/en/latest/creating-new/create/cmake.html

File: docs/packages/pkg/termcolor.rst (hunk 1 above)
Comment (norbertwenzel) [reply docs/packages/pkg/termcolor.rst]: Removed. Thx.

File: docs/packages/pkg/termcolor.rst (hunk 3 above)
Comment (norbertwenzel) [reply docs/packages/pkg/termcolor.rst:12]: Copied. Thx.

File: docs/packages/pkg/termcolor.rst (hunk 2 above)
Comment (norbertwenzel) [reply docs/packages/pkg/termcolor.rst]: Thx again.

**Pull request closed without merging**